src/builtins/Transformers.jl

```diff
@@ -2051,3 +2100,59 @@ julia> transform(mach, x)
 
 """
 UnivariateTimeTypeToContinuous
+
+"""
+$(MLJModelInterface.doc_header(InteractionTransformer))
+
+Generates all polynomial interaction terms up to the given order for the subset of chosen columns. 
+Any column that contains elements with scitype `<:Infinite` is a valid basis to generate interactions. 
+If `colnames` is not specified, all such columns with scitype `<:Infinite` in the table are used as a basis.
+
+
```

Comment:
A few small details missing in the docstring, but I can add those before merge.

Reply:
Allright, I'll let you adress that then